utils/src/ast-grep/resolve-binding-path.ts

```diff
@@ -0,0 +1,226 @@
+import type { TypesMap } from "@codemod.com/jssg-types/main";
+import type { Kinds } from "@ast-grep/napi/types/staticTypes.js";
```

Comment:
what if we only use `@codemod.com/jssg-types` ?

Reply:
my bad, replaced by `@ast-grep/napi` imports